README.md

```diff
@@ -10,7 +10,7 @@ overriding the language.content_setting configuration.
 This module provides a sub-module (oe_content_entity) with a set of Corporate content entities to be used.
 Check out the [README](/modules/oe_content_entity/README.md) of the module.
 
-The module sets the "Content Owner" field value automatically when new content is created. The default "Content Owner" field value can be set in the "Basic site settings" administrative page. This functionality is available only if [OpenEuropa Corporate Site Information](https://github.com/openeuropa/oe_corporate_site_info) is enabled.
+The module sets the "Authors" field value automatically when new content is created. The default "Authors" field value (reference to "Corporate body" sub-entity) can be set in the "Basic site settings" administrative page. This functionality is available only if [OpenEuropa Corporate Site Information](https://github.com/openeuropa/oe_corporate_site_info) is enabled.
```

Comment:
We shouldn't replace "Content owner" for "Authors", both are set automatically when creating nodes.

Reply:
My bad. Misunderstood requirements. Indeed, we shouldn't touch old content owner fields.